### ISSUE TITLE
add percentage input

### ISSUE THREAD

**Issue by Poltuu** [LuccaSA/lucca-ui#20]

**Comment by bpaugam** [comment]
```` html
<div class="lui has addon right input">
    <i class="lui addon">%</i>
    <input type="text" value="100" />
</div>
````

**Comment by Poltuu** [comment]
Does not properly work.
Let's keep this input in mind

**Comment by lucienbertin** [comment]
@Poltuu: i'm gonna need an attribute format to know how to format `10%`
i was thinking:
 - format="X%" - `10%` -> `10`
 - format="0.X" - `10%` -> `0.1`
 - format="1.X" - `10%` -> `1.1`
what do you think

**Comment by lucienbertin** [comment]
a first draft is available in this [page](http://luccasa.github.io/lucca-ui/directives.html)

this draft is shipped in version 1.1.1
